### ISSUE TITLE
shareapi_allow_share_dialog_user_enumeration call is wrong - no autocompletion if adding members

### ISSUE THREAD

**Issue by alx-tuilmenau** [nextcloud/circles#316]
Hi,

in Controller/MembersController.php in function searchGlobal():

Circles searchs value for "shareapi_allow_share_dialog_user_enumeration" within "circles" (via configService->getAppValue) , but the setting is in "core". So it's always empty.

Ugly fix with adding this value manually on command line within "circles":
`occ config:app:set circles  shareapi_allow_share_dialog_user_enumeration --value=yes`
works, but is not the final solution.

PS: Maybe it's also possible to honor
```
sharing.minSearchStringLength
sharing.maxAutocompleteResults
```
settings ("system") also. But for this you need to change the API of the Search-Classes to pass the limit.

**Comment by alx-tuilmenau** [comment]
@daita My issue report is for 0.16.7. But the fix in 0.16.8 comparing to "no" does not fix the problem (now autocomplete is always on). You may add a function to the ConfigService (or add a parameter to getAppValue) to allow getting the setting from 'core' instead of 'circles' ($this->appName).

**Comment by alx-tuilmenau** [comment]
Already fixed in 0.16.8
